nodejs/awsx/apigateway/api.ts

```diff
@@ -581,6 +624,74 @@ function addEventHandlerRouteToSwaggerSpec(
     }
 }
 
+function addAuthorizersToSwagger(swagger: SwaggerSpec, authorizers: authorizer.LambdaAuthorizerDefinition[]): string[] {
+    const authNames: string[] = [];
+    swagger["securityDefinitions"] = swagger["securityDefinitions"] || {};
+
+    for (const auth of authorizers) {
+        const suffix = Object.keys(swagger["securityDefinitions"]).length;
```

Comment:
nit: i honestly think it would be easier to understand if this was just:

```ts
for (let i = 0; i < authorizers.length; i++) {
    const auth = authorizers[i];
    // and then just use 'i' as the suffix
```

Reply:
The problem here is that multiple routes can add authorizers.So if you choose to not name the authorizers in 2 different routes you would end up w/ two different "authorizer-0". I am going to change the name to have a random 5 character suffix.

Reply:
ahhhhhhhh.  i missed that.  actually in that case your technique works well.  might be worth a comment... but it's probably ok without one.